code/modules/mod/mod_paint.dm

```diff
@@ -144,8 +144,11 @@
 		balloon_alert(user, "no alternate skins!")
 		return
 	var/list/skins = list()
-	for(var/mod_skin in mod.theme.skins)
-		skins[mod_skin] = image(icon = mod.icon, icon_state = "[mod_skin]-control")
+	for(var/mod_skin_name in mod.theme.skins)
+		var/list/mod_skin = mod.theme.skins[mod_skin_name]
+		skins[mod_skin_name] = image(icon = mod.icon, icon_state = "[mod_skin_name]-control")
+		if(mod_skin[MOD_ICON_OVERRIDE])
+			skins[mod_skin_name] = image(icon = mod_skin[MOD_ICON_OVERRIDE], icon_state = "[mod_skin_name]-control")
```

Comment:
```suggestion
		skins[mod_skin_name] = image(icon = mod_skin[MOD_ICON_OVERRIDE] || mod.icon, icon_state = "[mod_skin_name]-control")
```

Reply:
this should work

Reply:
but test it

Reply:
![image](https://github.com/tgstation/tgstation/assets/8430839/b7a6b9b0-4191-4c15-b0d4-6843a3da2ede)

Sure thing. Thanks!